controllers/bounce_processes_test.go

```diff
@@ -434,34 +434,6 @@ var _ = Describe("bounceProcesses", func() {
 				Expect(pendingUpgrades).NotTo(BeEmpty())
 			})
 		})
-
-		When("one process is already upgraded for a version compatible upgrade", func() {
```

Comment:
Why this test is getting dropped? Instead you should probably modify it to verify that upgrade completes if a process has already been upgraded?

Reply:
The reason for dropping this test is that we are not restarting processes with the kill command during a patch upgrade. We will do a rolling recreation of all Pods (except the transaction system Pods, those will be replaced to reduce the number of recoveries).